workspaces/vscode-java/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM prairielearn/workspace-vscode-base:latest
+
+USER root
+RUN apt-get update && apt-get install -y openjdk-21-jdk-headless && apt-get clean
```

Comment:
Inspired by #11221, can we add `--no-install-recommends` and `rm -rf /var/lib/apt/lists/*` to minimize the image size? For future comparison, before those potential changes, the images were ~555MB.